packages/content/src/Application/ContentResolver/Resolver/ResolverInterface.php

```diff
@@ -25,4 +25,11 @@ interface ResolverInterface
      * @param DimensionContentInterface<T> $dimensionContent
      */
     public function resolve(DimensionContentInterface $dimensionContent): ContentView;
+
+    /**
+     * @template T of ContentRichEntityInterface
+     *
+     * @param DimensionContentInterface<T> $dimensionContent
+     */
+    public function supports(DimensionContentInterface $dimensionContent): bool;
```

Comment:
Not really a fan of supports methods, as we don't use it in other cases in the content bundle and do a early return there. The supports methods ends up in a double check of the type because we always need todo checks inside the called method also.

What do you think about make the resolver nullable return type instead e.g.:

```php
public function resolve(DimensionContentInterface $dimensionContent): ?ContentView
```

Reply:
I was also not really sure, both options have some cons, e.g. null checking. But I don't have a strong opinion on this, so if you prefer it to be nullable, I can refactor it :)

Reply:
as discussed we will go with `?ContentView`